website/source/docs/concepts/seal.html.md

```diff
@@ -93,17 +93,19 @@ For a list of examples and supported providers, please see the
 
 The seal can be migrated from Shamir keys to Auto Unseal and vice versa.
 
-  ~> **NOTE**: This is not currently supported when using replication. While
-  the primary can be migrated without issue, the secondaries, depending on
+  ~> **NOTE**: This is not currently supported when using Vault Enterprise Replication.
```

Comment:
I wanted to be specific here, as replication is not a feature of Vault Open Source so this could be misleading if not familiar with Enterprise.